src/js/tech/tech.js

```diff
@@ -531,20 +548,33 @@ class Tech extends Component {
    *
    * @param {Object} options The object should contain values for
    * kind, language, label and src (location of the WebVTT file)
+   * @param {Boolean} manualCleanup if set to false, the TextTrack will be
```

Comment:
also you should add an `@depricated` tag to the bottom stating that "The `manualCleanup` parameter will be removed in favor of always doing automatic cleanup in the future."